e2e/atlas/datalake_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build e2e atlas,datalake
```

Comment:
you could leave this on the generic group if you want, the ones we have split out of generic is usually because they depend on a cluster being healthy which takes time (10-15 mins) so we put them apart to avoid long waits on simpler tasks

Reply:
cool! Will do

Reply:
I did a PR early adding comments to our evg file about this, let me know if there's something else we can/should document